src/components/com_tjcertificate/site/views/certificates/tmpl/my.php

```diff
@@ -92,9 +115,16 @@
 								<tr class="row <?php echo $i % 2; ?>" sortable-group-id="<?php echo $item->id; ?>">
 								<td class="has-context">
 									<div class="pull-left break-word">
+										<?php if (!$item->is_external) {?>
 										<a href="<?php echo TJCERT::Certificate($item->id)->getUrl('',false); ?>">
```

Comment:
try updating the same function for getting external certificate URL as well

Reply:
done